pyccl/pk2d.py

```diff
@@ -147,15 +188,21 @@ def from_model(cls, cosmo, model):
         """`Pk2D` constructor returning the power spectrum associated with
         a given numerical model.
 
-        Args:
-            cosmo (:class:`~pyccl.core.Cosmology`): A Cosmology object.
-            model (:obj:`str`): model to use. Three models allowed:
-                `'bbks'` (Bardeen et al. ApJ 304 (1986) 15).
-                `'eisenstein_hu'` (Eisenstein & Hu astro-ph/9709112).
-                `'eisenstein_hu_nowiggles'` (Eisenstein & Hu astro-ph/9709112).
-                `'emu'` (arXiv:1508.02654).
-        """
-        pk2d = Pk2D(empty=True)
+        Arguments:
+            cosmo (:class:`~pyccl.core.Cosmology`)
+                A Cosmology object.
+            model (:obj:`str`)
+                model to use. These models allowed:
+                  - `'bbks'` (Bardeen et al. ApJ 304 (1986) 15)
+                  - `'eisenstein_hu'` (Eisenstein & Hu astro-ph/9709112)
+                  - `'eisenstein_hu_nowiggles'` (Eisenstein & Hu astro-ph/9709112)
+                  - `'emu'` (arXiv:1508.02654).
+        Returns:
+            :class:`~pyccl.pk2d.Pk2D`
+                The power spectrum of the input model.
+        """  # noqa E501
```

Comment:
Don't worry about fixing the docstring style for now, even this won't show right. I've opened a PR for that, but before we do anything we have to read numpy's style guide.